homeassistant/components/statistics/sensor.py

```diff
@@ -86,6 +86,13 @@
     "for further details: "
     "https://www.home-assistant.io/integrations/statistics/"
 )
+DEPRECATION_WARNING_SIZE = (
+    "The configuration of either 'sampling_size' or 'max_age' will become "
+    "mandatory in a future release of the statistics integration. Please "
```

Comment:
I think we can add a target version where the behavior changes already now, for example like this for a one-month notice period:
```suggestion
    "mandatory in a Home Assistant Core 2022.7. Please "
```

Reply:
Good idea. Done.